locales/bn.json

```diff
@@ -411,6 +419,26 @@
     "message": "কমান্ড প্রম্পটে এই কমান্ডটি চালিয়ে নিজেকে রিমুভ করতে হবে:",
     "description": "Call to action, followed by a computer command"
   },
+  "modals.bucketinfo.btn1": {
+    "message": "এই বালতি ফর্ক করুন",
+    "description": "Button text"
+  },
+  "modals.bucketinfo.btn1p": {
+    "message": "এই প্রকল্পটি ফর্ক করুন",
+    "description": "Button text"
+  },
+  "modals.bucketinfo.label1": {
+    "message": "তৈরি করা হয়েছে:",
+    "description": "Label, followed by a date and time"
+  },
+  "modals.bucketinfo.label2": {
+    "message": "থেকে কাঁটা:",
+    "description": "Label, followed by a link"
+  },
+  "modals.bucketinfo.p1": {
+    "message": "এই বালতি বা প্রকল্পের নিজস্ব অনুলিপি তৈরি করতে নিচে ক্লিক করুন।",
```

Comment:
Change this to this :arrow_right:  `"message": "এই বাকেটটির বা প্রকল্পের নিজস্ব অনুলিপি তৈরি করতে নিচে ক্লিক করুন।"`